modules/upgrade/pages/upgrade-script-example.adoc

```diff
@@ -62,3 +67,296 @@ Use this script as the basis for an upgrade from SLE{nbsp}12{nbsp}SP4 to SLE{nbs
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <ask_on_error config:type="boolean">true</ask_on_error>
+        <media_url>http://$redhat_management_server/ks/dist/child/sle-module-suse-manager-proxy-4.0-updates-x86_64/$distrotree</media_url>
+        <name>sle-module-suse-manager-proxy-4.0-updates-x86_64</name>
+        <product>SUSE Manager Proxy 4.0</product>
+        <product_dir>/</product_dir>
+      </listentry>
+     </add_on_products>
+  </add-on>
+  <general>
+      $SNIPPET('spacewalk/sles_no_signature_checks')
+    <mode><confirm config:type="boolean">false</confirm></mode>
+  </general>
+  <upgrade>
+    <only_installed_packages config:type="boolean">false</only_installed_packages>
+    <stop_on_solver_conflict config:type="boolean">true</stop_on_solver_conflict>
+  </upgrade>
+  <backup>
+    <sysconfig config:type="boolean">true</sysconfig>
+    <modified config:type="boolean">true</modified>
+    <remove_old config:type="boolean">false</remove_old>
+  </backup>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+    <start_immediately config:type="boolean">true</start_immediately>
+  </networking>
+  <software>
+    <image/>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <instsource/>
+    <patterns config:type="list">
+      <pattern>suma_proxy</pattern>
+    </patterns>
+    <products config:type="list">
+      <listentry>SUSE-Manager-Proxy</listentry>
+    </products>
+  </software>
+  <scripts>
+    <chroot-scripts config:type="list">
+      <script>
+        <filename>cleanup.sh</filename>
+        <chrooted config:type="boolean">true</chrooted>
+        <interpreter>shell</interpreter>
+        <source>
+        <![CDATA[
+#!/bin/sh
+         sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES wsgi
+         if [ -e /etc/rhn/rhn.conf.rpmsave ]; then
+             cp /etc/rhn/rhn.conf.rpmsave /etc/rhn/rhn.conf
+         fi
+         zypper rr --all
+         ]]>
+        </source>
+      </script>
+    </chroot-scripts>
+  </scripts>
+</profile>
+----
+
+
+
+== Upgrading an SAP Client from SLE{nbsp}15 to SLE{nbsp}15.1
+
+Use this script as the basis for an upgrade from SLE{nbsp}15 to SLE{nbsp}15.1.
+
+.Example: Script for Upgrading an SAP Client from SLE{nbsp}12{nbsp}SP4 to SLE{nbsp}15
+
+If you use this profile, when the uploaded profile requires variables to be set, navigate to menu:Systems[Autoinstallation > Profiles], select the profile to edit, and navigate to the [guimenu]``Variables`` tab.
+Specify the required variables, using this format:
+
+----
+<key>=<value>
+----
+
+For SAP installations, the ``$redhat_management_server`` variable will be set automatically and does not need to be defined.
+
+This example uses the ``$distrotree`` variable, which must be defined as the distribution label.
+Set this variable as:
+
+----
+distrotree=sles_sap15sp1-x86_64
+----
```

Comment:
Should we put this in a general section? It is valid for all autoinstallations.

We should also say something about the registration at the end. Depending on what mechanism you use (salt vs. traditional) you need different snippets (minion_script vs sles_register_script).